### DISCUSSION TITLE
Creation Time in GoPro Video

### DISCUSSION THREAD

**Discussion by DaveBrillhart** [gopro/labs#490] [General]
My Use Case is using the Quick Capture to automatically start recording on my Hero 11 Black when I turn on the camera. I take a series of short videos at various locations during a multi-hour cycling ride. So, there isn't enough time, usually, for the camera to get a GPS Lock.

What I need is for those videos to have an accurate Media Creation Time.

I can make sure to sync the Hero 11 with my phone. But I don't know if the Hero remembers the right time after a power off and back on, so each Quick Capture video has the correct Creation Time?

I also don't know if the camera, if it does get a GPS Lock (and time) if it retroactively updates the Creation Time to the right value?

I think there is Lab Firmware (Precision Time QR Code) that may help. I don't know if that is a once and done setting, or I have to scan in a QR code before each session?

Any tips would really be appreciated.

**Comment by dnewman-gpsw** [comment]
If you keep the battery in the camera, creation time is accurate.  At worst it might drift a second a day.  If you use !MSYNC=1, the camera's GPS will remove this drift whenever GPS lock is established.  Also if you use https://gopro.github.io/labs/control/precisiontime/ just once every few days your creation times will be accurate to within a second.

**Reply by DaveBrillhart** [reply]
Thanks! What I've done is load the latest (Aug, 2023) Labs Firmware. Then I pointed my camera at the Precision Time QR Code, which it registered. I also downloaded the iOS Labs Mobile App, and configured the settings I want PLUS checked the Update Time Box. That seems to be the way to go. So I think before each session, I'll turn on the camera and sync with the QR code on the Labs mobile app. Then all the clips I make that day should be good to go. THANK YOU.
